src/app/dynamic_server/AccessControl.h

```diff
@@ -19,6 +19,7 @@
 
 constexpr chip::EndpointId kOtaProviderDynamicEndpointId     = 0;
 constexpr chip::EndpointId kWebRTCRequesterDynamicEndpointId = 1;
+constexpr chip::EndpointId kWebRTCProviderDynamicEndpointId  = 1;
```

Comment:
Actually, this really confuses me: why are all users of DynamicDispatcher suddenly doing WebRTC?  I know some of it was pre-existing, but that part looks wrong too.

Reply:
My point is that these are not the only consumers of DynamicDispatcher, and the way this is being hacked into the DynamicDispatcher is broken for all the other consumers.

Reply:
Okay, Thank you for sharing the information. Actually, we are going to provide intercom types of device. "https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/Intercom.adoc#cluster-requirements" and WEBRTC 1.5 TCs "https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/webrtc_transport.adoc#tc-webrtc-1-5-validate-that-the-camera-dut-can-start-a-webrtc-session-by-issuing-an-provideoffer-command-provisional". We will discuss good way and let you check your feedback. Appreciated your time.